### PR TITLE
use file leak detector lib 1.10

### PR DESCRIPTION
**Pull request by denisj44**
To support pipe FileDescriptor with the latest file leak detector library.
@reviewbybees 

Can be tested by executing the following groovy script after started with the "trace" option:
> import java.nio.channels.Pipe;
> final Pipe s = Pipe.open();
> //s.sink().close();
> s.source().close();
> println("done");

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,17 +3,22 @@
...
-    <version>1.466</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>3.2</version>
   </parent>
 
   <groupId>com.cloudbees.jenkins.plugins</groupId>
   <artifactId>file-leak-detector</artifactId>
   <name>CloudBees File Leak Detector Plugin</name>
   <description>Monitors and detects file handle leaks in Jenkins</description>
-  <version>1.6-SNAPSHOT</version>
+  <version>1.6</version>
```
Comment (oleg-nenashev) [inline pom.xml]: hmm? you do not need to do it in Maven projects

File: file-leak-detector.iml [hunk 2]
```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
```
Comment (oleg-nenashev) [inline file-leak-detector.iml]: Please remove the IDEA file from the repo

**Comment by denisj44** [comment]
@oleg-nenashev Recommendations followed. Thank you.

File: pom.xml [hunk 3]
```diff
@@ -3,17 +3,22 @@
...
   <packaging>hpi</packaging>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/File+Leak+Detector+Plugin</url>
```
Comment (oleg-nenashev) [inline pom.xml]: "wiki.jenkins.io" while you are around

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Comment by dwnusbaum** [comment]
@denisj44 From some manual testing of this PR vs master, it looks like detection of regular files is broken, and only open pipes are being detected. Have you verified that regular files are still detected normally?

**Pull request merged**